include/experimental/__p0009_bits/extents.hpp

```diff
@@ -614,5 +615,65 @@ static
 #endif
 constexpr bool __is_extents_v = __is_extents<T>::value;
 
+template<class UserIndexType, class IndexType>
+constexpr void check_lower_bound(const UserIndexType& user_index,
+                                 const IndexType& /* current_extent */,
+                                 std::true_type) /* is_signed */
+{
+#if defined(NDEBUG)
+  (void) user_index;
+#else
+  assert(static_cast<IndexType>(user_index) >= 0);
+#endif
```

Comment:
```suggestion
#if defined(NDEBUG)
  (void) user_index;
#endif
  assert(static_cast<IndexType>(user_index) >= 0);
```
same comment everywhere.
(Rational: to mitigate the risk for a misspelled macro)
Alternatively drop the macro guards altogether and just unconditionally cast to void.

Reply:
Good point -- I could also use `[[maybe_unused]]` here but I didn't want to make life harder for users who want a C++14 back-port.

Reply:
Fixed; thanks!